tableau_utilities/tableau_file/tableau_file_objects.py

```diff
@@ -753,9 +758,12 @@ class ParentConnection(TableauFileObject):
     cols: TableauFileObjects[MappingCol] = None
     refresh: Refresh = None
     metadata_records: TableauFileObjects[MetadataRecord] = None
+    default_settings: str = None
```

Comment:
I was getting an error on default_settings until I added this line.

Here's the error 
```
Traceback (most recent call last):
  File "/Users/jayrosenthal/.pyenv/versions/3.10.3/bin/tableau_utilities", line 33, in <module>
    sys.exit(load_entry_point('tableau-utilities==2.0.0', 'console_scripts', 'tableau_utilities')())
  File "/Users/jayrosenthal/.pyenv/versions/3.10.3/lib/python3.10/site-packages/tableau_utilities/scripts/cli.py", line 254, in main
    args.func(args)
  File "/Users/jayrosenthal/.pyenv/versions/3.10.3/lib/python3.10/site-packages/tableau_utilities/scripts/gen_config.py", line 380, in generate_config
    metadata_record_columns = get_metadata_record_columns(datasource_name, datasource_path, debugging_logs)
  File "/Users/jayrosenthal/.pyenv/versions/3.10.3/lib/python3.10/site-packages/tableau_utilities/scripts/gen_config.py", line 94, in get_metadata_record_columns
    metadata_records = [c.dict() for c in Datasource(datasource_path).connection.metadata_records]
  File "/Users/jayrosenthal/.pyenv/versions/3.10.3/lib/python3.10/site-packages/tableau_utilities/tableau_file/tableau_file.py", line 114, in __init__
    self.extract: tfo.Extract = tfo.Extract(**self.__get_section('extract')[0])
  File "<string>", line 9, in __init__
  File "/Users/jayrosenthal/.pyenv/versions/3.10.3/lib/python3.10/site-packages/tableau_utilities/tableau_file/tableau_file_objects.py", line 863, in __post_init__
    self.connection = ParentConnection(**transform_tableau_object(self.connection))
TypeError: ParentConnection.__init__() got an unexpected keyword argument 'default_settings
```

Adding this line makes things work.  here's the connection string with that parameter.  

```
      <extract _.fcp.ObjectModelExtractV2.true...object-id='' count='-1' enabled='true' units='records'>
        <connection access_mode='readonly' authentication='auth-none' author-locale='en_US' class='hyper' dbname='/Users/jayrosenthal/Documents/My Tableau Repository/Datasources/DIM_INFRASTRUCTURE_COSTS (INFRASTRUCTURE.DIM_INFRASTRUCTURE_COSTS) (INFRASTRUCTURE).hyper' default-settings='yes' schema='Extract' sslmode='' tablename='Extract' update-time='01/06/2023 12:00:35 AM' username='tableau_internal_user'>
```

Am I fixing this correctly here?